addons/stock/views/stock_quant_views.xml

```diff
@@ -373,6 +335,18 @@
         </field>
     </record>
 
+    <record model="ir.actions.server" id="action_view_set_quants_tree">
```

Comment:
The issue with that, is now it's available in inventory report and trigger the action won't do any change in the view and don't return information to cust.

Reply:
yep nice catch. Need to discuss with Thomas